esdl-singleuser-julia-nb/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/esdc/singleuser-python-nb:0.5.7
```

Comment:
I am not sure which version of the python notebook to use here

Reply:
I will test whether it works. We shoud use latest but I seem to remember that there was an issue with our installation.

Which reminds me that we considered to switch to the datascience notebook entirely. I would have time now to work on that.

Reply:
I have tested it locally with latest and it seemed to work, but I remember there was an edge case with some shared library, it will be in the forum somewhere. 

> Which reminds me that we considered to switch to the datascience notebook entirely. I would have time now to work on that.

That sound great. Let me know when I can help with the julia-specific installation, or you just start with the python packages and I will add the Julia stuff on top.